src/ui/RealmBrowser/Content/CreateObjectDialog/types/TypeControl.tsx

```diff
@@ -123,12 +124,12 @@ export const TypeControl = ({
       />
     );
   } else if (property.type === 'object') {
-    // TODO: implement
-    if (isEmbeddedType(property.objectType!)) {
+    if (isEmbeddedType(property.objectType)) {
       return (
         <DefaultControl
+          type="info"
           property={property}
-          message="Creation of embedded objects is not yet supported"
+          message="Add an embedded object from the grid view, once created."
```

Comment:
I don't think we're referring to this as the "grid" in any other user facing text. I don't have a strong feeling about this, but suggest using table instead.

```suggestion
          message="Embedded objects can be created from the table, once this object has been created."
```